README.md

```diff
@@ -6,6 +6,12 @@ PHP SDK for the Wizaplace e-commerce API: https://sandbox.wizaplace.com/api/v1/d
 composer require wizaplace/sdk
 ```
 
+You can also use Vagrant environment :
```

Comment:
Mauvaise section. `##Install` est là pour les utilisateurs.

Pour les devs,  ca se passe dans `## Running linters and tests`